### PR TITLE
refine rich text and other i18n

### PR DESCRIPTION
**Pull request by knoxHuang**
Re: #

### Changelog

* refine rich text and other i18n

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: cocos/2d/components/rich-text.ts [hunk 1]
```diff
@@ -23,7 +23,18 @@
  THE SOFTWARE.
 */
 
-import { ccclass, executeInEditMode, executionOrder, help, menu, tooltip, multiline, type, displayOrder, serializable } from 'cc.decorator';
+import {
```
Comment (wuzhiming) [inline cocos/2d/components/rich-text.ts]: 我看引擎都是单行引用，这边需要也改一下

**Review by wuzhiming — APPROVED** [review]

File: cocos/2d/components/rich-text.ts [hunk 2]
```diff
@@ -162,7 +162,6 @@ export class RichText extends Component {
      * 富文本显示的文本内容。
      */
     @multiline
-    @tooltip('i18n:richtext.string')
```
Comment (minggo) [inline cocos/2d/components/rich-text.ts:165]: Why remove it?

File: cocos/2d/framework/ui-renderer.ts [hunk 3]
```diff
@@ -168,7 +168,6 @@ export class UIRenderer extends Renderer {
      */
     @type(Material)
     @displayOrder(0)
-    @displayName('CustomMaterial')
```
Comment (minggo) [inline cocos/2d/framework/ui-renderer.ts:171]: Why remove it?

File: editor/i18n/zh/assets.js [hunk 4]
```diff
@@ -152,6 +152,7 @@ module.exports = {
             isRGBE: '作为 RGBE 格式',
             isRGBETip: '作为 RGBE 格式',
             flipGreenChannel: '翻转绿色通道',
+            flipGreenChannelTip: '是否翻转绿通道',
```
Comment (minggo) [inline editor/i18n/zh/assets.js]: 绿通道 -> 绿色通道

**Comment by knoxHuang** [comment]
https://github.com/cocos/3d-tasks/issues/18563 

the ui performance of the synchronous cache mode should remain consistent with the label.

File: cocos/2d/components/rich-text.ts (hunk 2 above)
Comment (knoxHuang) [reply cocos/2d/components/rich-text.ts:165]: use automatic addition of tooltip

code: ./app/builtin/scene/source/script/export/dump/encode.ts

image

File: cocos/2d/framework/ui-renderer.ts (hunk 3 above)
Comment (knoxHuang) [reply cocos/2d/framework/ui-renderer.ts:171]: handle with a custom approach to adding i18n

code: ./app/builtin/scene/source/script/export/dump/encode.ts

image

File: editor/i18n/zh/assets.js (hunk 4 above)
Comment (knoxHuang) [reply editor/i18n/zh/assets.js]: done

**Review by minggo — APPROVED** [review]

**Pull request merged**